combat-trainer.lic

```diff
@@ -369,7 +373,29 @@ class LootProcess
       echo 'Detected harvest messaging' if $debug_mode_ct
       @last_ritual = ritual
       waitrt?
-      bput('drop material', 'you discard it')
```

Comment:
Given the size of this `case`, it might be worth pulling the code out into a helper.

You could name it something like `store_material?` and drop if it returns false.